### DISCUSSION TITLE
adrs and outp do not match

### DISCUSSION THREAD

**Discussion by da-doodoo** [hlorenzi/customasm#208] [Q&A] [answered]
Everytime I try compiling some assembly language in logisim, the adresses are always misaligned (incremented by 1), how do I fix this??
![image](https://github.com/user-attachments/assets/a97a9f03-3e47-4aaa-bb72-ed5f6fb7afbb)
![image](https://github.com/user-attachments/assets/d342fc08-f7d8-486c-b77d-eacf9a4963ee)

**Comment by hlorenzi** [comment]
If you have used a `#bankdef`, is its `#outp` field zero?

Could you post your source code (or a small example) so I can check if there's some other issue that's causing this?

**Reply by da-doodoo** [reply]
Im sorry for the EXTREMELYYYYYYYY late reply, but here you go: [https://filebin.net/6eudgqx5j5w61t2o](url)
BTW thanks for creating such a easy-to-use software for custom ISRs. It saved me and probably countless others a LOT of time we would have otherwise spent creating entirely custom parsers.

**Reply by da-doodoo** [reply, marked as answer]
I set outp to 0 it worked thanks!!

**Reply by hlorenzi** [reply]
It's great that you were able to get it working!
Although I'm still suspecting this might be a bug in the assembler, going from your code. `#outp` should be zero by default, and you shouldn't need to do that. I'll have to look into it.
